saga-web/src/main/resources/saga-frontend/src/app/@core/core.module.ts

```diff
@@ -0,0 +1,75 @@
+import { ModuleWithProviders, NgModule, Optional, SkipSelf } from '@angular/core';
```

Comment:
We need to apply the apache License header here.

Reply:
Updated the files with appropriate license information.